jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -54,26 +54,53 @@ define([
     PolymerSupport.init();
 
     // dashboard-actions depends on requirejs text plugin
-    require(['./dashboard-actions'], function(DashboardActions) {
+    require(['./dashboard-actions', './dashboard-metadata'], function(DashboardActions, Metadata) {
```

Comment:
When we have to start supporting 2 metadata formats potentially, the old one + the new official Jupyter one, then it might be nice to have this separation.